PopupMenu.js

```diff
@@ -0,0 +1,131 @@
+import styled from "styled-components";
+import { Icon } from "@iconify/react";
+import { useState, useEffect } from "react";
+import { useStore } from "../hooks/useStore";
+import CollectionItem from "./CollectionItem";
+export default function PopupMenu({ popUp, setPopUp, article }) {
+  const [editMode, setEditMode] = useState(false);
+  const addCollection = useStore((state) => state.addCollection);
+  const collections = useStore((state) => state.collections);
+  const add2Collection = useStore((state) => state.add2Collection);
+  const removeCollection = useStore((state) => state.removeCollection);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formdata = new FormData(event.target);
+    const data = Object.fromEntries(formdata);
+    addCollection({
+      id: Math.random().toString(36).substring(2),
+      name: data.name,
+      description: data.description,
+      articles: [],
+    });
+
+    setEditMode(false);
+  }
+
+  function onSelectedCollection(id) {
```

Comment:
vielleicht könntest du hier noch eine Abfrage einbauen, ob das Item schon in der collection ist und sonst einen Altert ausschmeißen? Aktuell kann man Artikel mehrfach in die Collections aufnehmen

Reply:
yes du hast recht, danke schön! Momentan ist im Hintergrund schon gecheckt ob ein Article schon in der Collection befindet. Allerdings ist ein Alert noch gebraucht, auch wie im Spotify :).
Ich hab gerade gemerkt dass man auch bei neuer Hinzufüge einer neuen Collection auch kontrollieren muss, ob die Collection schon angelegt wurde. Momentan ist es nach id gecheckt, aber das Richtige wäre nach Name.